test_public_network.py

```diff
@@ -67,7 +67,7 @@ def test_public_network_connectivity_on_all_images(server):
         server.ping(address, count=3, interval=0.5)
 
 
-def test_public_network_mtu(server):
+def test_public_network_mtu(server, image):
```

Comment:
Why has `image` been added here? It seems unused.

Reply:
That seems to be a leftover, removed.